src/components/table/examples/header-menu.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from '@stencil/core';
+import { TableComponent } from '../table.types';
+
+@Component({
+    tag: 'limel-example-header-menu',
+    shadow: true,
+})
+export class HeaderMenu implements TableComponent {
+    @Prop()
+    public data: any;
```

Comment:
We should not use `any` here. It makes it very hard for the reader to understand what it is used for